prdoc/pr_5679.prdoc

```diff
@@ -0,0 +1,42 @@
+title: "Switch to new `CandidateReceipt` primitives"
+
+doc:
+  - audience: Node Dev
+    description: |
+      This change is just plumbing work and updates all crate interfaces to use the new primitives.
+      Id doesn't alter any functionality and is required before implementing RFC103 on the 
+      node side. 
+
+crates:
+  - name: polkadot-primitives
+    bump: minor
+  - name: polkadot-runtime-parachains
+    bump: minor
+  - name: cumulus-client-consensus-common
+    bump: major
+  - name: cumulus-client-pov-recovery
+    bump: major
```

Comment:
why are these two major? I think they should be none

actually, why is everything a major bump? :D

Reply:
I've switched the subsystem messages and cumulus traits to new primtives so this breaks things.

Also I'm missing many crates here and the check fails.